boards/cc2538dk/include/periph_conf.h

```diff
@@ -151,6 +151,21 @@ static const periph_spi_conf_t spi_config[SPI_NUMOF] = {
 /** @} */
 
 /**
+ * @name ADC configuration
+ * @{
+ */
+#define ADC_NUMOF           (1)
+#define ADC_ALS_PIN         6
```

Comment:
This applies to all the periph_conf files that you adapted: Isn't there a way to be more consistent with the GPIO naming scheme? I don't find these names somehow speaking or at least self explaining. I won't know which pin to use without deeper research on that.